docs/content/install.sh

```diff
@@ -1,4 +1,5 @@
-#!/bin/sh
+#!/bin/bash
```

Comment:
To be consistent with the other scripts used by rclone, you should change the path to bash to
```sh
#!/usr/bin/env bash
```